cmd/otelcontribcol/main.go

```diff
@@ -5,7 +5,6 @@ package main
 
 import (
 	"log"
-
```

Comment:
This file is autogenerated and should not have changes of this type, can you revert the changes on `main.go` and in `main_windows.go`?